.github/workflows/main.yml

```diff
@@ -686,6 +687,21 @@ jobs:
         id: test_notebook_vscode
         if: matrix.test-suite == 'notebook' || matrix.test-suite == 'conda'
 
+      - name: Run Remote Native Notebook with VSCode & Jupyter
```

Comment:
Could do this by just having a step that sets the VSC_JUPYTER_REMOTE_NATIVE_TEST environment globally if the matrix is remoteNotebook? Then you could share the actual running of the test

Reply:
Done.